packages/cli/src/evaluation.ee/test-runner/test-runner.service.ee.ts

```diff
@@ -259,16 +259,11 @@ export class TestRunnerService {
 			throw new TestRunError('EVALUATION_TRIGGER_NOT_FOUND');
 		}
 
-		// Initialize the input data for dataset trigger
-		// Provide a flag indicating that we want to get the whole dataset
-		const nodeExecutionStack: IExecuteData[] = [];
-		nodeExecutionStack.push({
-			node: triggerNode,
-			data: {
-				main: [[{ json: { requestDataset: true } }]],
-			},
-			source: null,
-		});
+		// Call custom operation to fetch the whole dataset
+		triggerNode.forceCustomOperation = {
+			resource: 'testRunner',
```

Comment:
Nitpick: it feels a bit unclear that resource is `testRunner` here. What happens is `[testRunner] --requests dataset--> 
[triggerNode]`.
Maybe resource: 'dataset', operation: 'request'?

Reply:
Thanks. That's a good point. Will rename them to be more clear and decouple the components better.